.vscode/settings.json

```diff
@@ -12,5 +12,10 @@
     "editor.insertSpaces": true,
     "editor.detectIndentation": false,
     "editor.tabSize": 4,
-    "typescript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false
+    "typescript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false,
+    "workbench.colorCustomizations": {
```

Comment:
@chrisdp  I forgot to mention this earlier, so I approved it so I could start the release. But this shouldn't have been committed.